snow/networking/handler/health.go

```diff
@@ -66,5 +66,13 @@ func (h *handler) getDisconnectedValidators() set.Set[ids.NodeID] {
 	connectedVdrs := h.peerTracker.ConnectedValidators()
 	// vdrs - connectedVdrs is equal to the disconnectedVdrs
 	vdrs.Difference(connectedVdrs)
-	return vdrs
+	return trimWeights(vdrs)
+}
+
+func trimWeights(weights set.Set[ids.NodeWeight]) set.Set[ids.NodeID] {
```

Comment:
should we call this `withoutWeights` or `dropWeights`or `nodeIDs`?

Reply:
`withoutWeights` sounds good!